pages/api/openPR.js

```diff
@@ -15,40 +15,199 @@ const GITHUB_BRANCH = process.env.GITHUB_BRANCH
   ? process.env.GITHUB_BRANCH
   : "main"; /* optional: defaults to default branch */
 
+// Return Project name
+function getProjectName(values) {
+  return values.name;
+}
+
+// Parse name of project prior to saving file
+function parseProjectName(values) {
+  return (
+    values.name
+      .normalize("NFD")
+      .toLowerCase()
+      .replace(/[\u0300-\u036f]/g, "")
+      .replace(/ /g, "-") + ".json"
+  );
+}
+
+// Create Github checkout branch for current submissions
+function createGithubCheckoutBranch(name) {
+  const checkoutBranch =
+    `${name}`.split(".").slice(0, -1).join(".") +
+    "-" +
+    (Math.random() * 10 ** 16).toString(36);
+  return checkoutBranch;
+}
+
+// Get SDG relevance info
+function getSDGRelevanceInfo(values, sdgNumber, evidenceText) {
+  //Loop through SDG array and parse SDG information
+  for (let i = 0; i < values.SDGs.length; i++) {
+    sdgNumber = parseInt(values.SDGs[i]);
+    evidenceText = "evidenceText".concat(sdgNumber);
+
+    values.SDGs[i] = {
+      SDGNumber: sdgNumber,
+      evidenceText: values[evidenceText],
+    };
+
+    delete values[evidenceText];
+  }
+  return values;
+}
+
+// Order fields e.g organizations
+function orderFields(values) {
+  let name, website, org_type;
+  // Order all entries for organizations
+  for (let i = 0; i < values.organizations.length; i++) {
+    name = values.organizations[i].name;
+    website = values.organizations[i].website;
+    org_type = values.organizations[i].org_type;
+
+    values.organizations[i] = {
+      name: name,
+      website: website,
+      org_type: org_type,
+    };
+  }
+  return values;
+}
+
+// Return multiple submission files including corresponding filepaths
+function getSubmissionFiles(values, myJSON) {
```

Comment:
Please use more descriptive variables, instead of `myJSON` consider `nomineeJSON`

Reply:
Resolved in commit b645ff6.